src/main.rs

```diff
@@ -554,11 +588,13 @@ mod test {
 
         let worker_port = 50051;
         let node_id = util::DEFAULT_NODE_NAME;
+        let ip_address = util::DEFAULT_NODE_IP;
```

Comment:
remove `DEFAULT_NODE_NAME` from `util` mod, just define it in this `test` mod

Reply:
Done